db/seeds/development-seeds.d/seeds.json

```diff
@@ -0,0 +1,93 @@
+{
+  "fixity_algs":[
+    { "name": "sha256" },
+    { "name": "md5" }
+  ],
+  "protocols":[
+    { "name": "rsync" }
+  ],
+  "storage_regions": [
+    { "name": "default" }
+  ],
+  "storage_types": [
+    { "name": "default" }
+  ],
+  "members": [],
+  "nodes": [
+    {
+      "namespace": "hathi",
+      "name": "HathiTrust",
+      "storage_region": "default",
+      "storage_type": "default",
+      "fixity_algs": ["md5","sha256"],
+      "protocols": ["rsync"],
+      "api_root": "http://127.0.0.1",
+      "private_auth_token": "hathi_token",
+      "auth_credential": "hathi_token",
+      "replicate_to_nodes": ["hathi", "sdr", "tdr", "chron", "aptrust"],
+      "replicate_from_nodes": ["hathi", "sdr", "tdr", "chron", "aptrust"],
+      "restore_to_nodes": ["hathi", "sdr", "tdr", "chron", "aptrust"],
+      "restore_from_nodes": ["hathi", "sdr", "tdr", "chron", "aptrust"]
+    },
```

Comment:
How do these `api_root` values (for all these nodes) intersect with the scripts for the local cluster?  (The scripts simply assume there is an array of nodes and assigns a localhost:port to each of them, in turn.  So the only difference between them is the port numbers.)